web/cobrands/sass/_base.scss

```diff
@@ -182,6 +182,11 @@ a {
   }
 }
 
+.meta-2 a[name] {
+    padding-top: 5em;
```

Comment:
Maybe worth leaving a comment that this magic `5em` number is roughly the height of `.nav-wrapper-2` (4em @ 16px) plus the border-top on `.nav-wrapper-2` (4px), plus the padding-top on `.item-list__item` (0.5em @ 16px).